### PR TITLE
Install the Product Offer Service Points Availability feature

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Tickets:
- https://spryker.atlassian.net/browse/CC-30833
- https://spryker.atlassian.net/browse/CC-30817
- https://spryker.atlassian.net/browse/CC-30224

Epics:
- https://spryker.atlassian.net/browse/CC-29686
- https://spryker.atlassian.net/browse/CC-29697
- https://spryker.atlassian.net/browse/CC-29674

### CODE REVIEW

**Review by kkicheglovspryker — DISMISSED** [review]

File: _includes/pbc/all/install-features/202311.0/install-the-shipment-product-offer-service-points-availability-feature.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+| TRANSFER                                                    | TYPE  | EVENT   | PATH                                                                                              |
+|-------------------------------------------------------------|-------|---------|---------------------------------------------------------------------------------------------------|
+| RestProductOfferServicePointAvailabilitiesRequestAttributes | class | created | src/Generated/Shared/Transfer/RestProductOfferServicePointAvailabilitiesRequestAttributesTransfer |
+| ProductOfferServicePointAvailabilityConditions              | class | created | src/Generated/Shared/Transfer/ProductOfferServicePointAvailabilityConditionsTransfer              |
+
+{% endinfo_block %}
+
+## 3) Set up behavior
+
+{% info_block warningBox "Verification" %}
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202311.0/install-the-shipment-product-offer-service-points-availability-feature.md:54]: There are no steps in this section. What are we verifying?

Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202311.0/install-the-shipment-product-offer-service-points-availability-feature.md:54]: @andriitserkovnyi, we are verifying that the filter can be used with the resource in Storefront API and there is only one step - `POST https://glue.mysprykershop.com/product-offer-service-point-availabilities`.

Comment (andriitserkovnyi) [reply _includes/pbc/all/install-features/202311.0/install-the-shipment-product-offer-service-points-availability-feature.md:54]: @romanhavrylko, I mean that there are no steps in this section besides verification.

Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202311.0/install-the-shipment-product-offer-service-points-availability-feature.md]: @andriitserkovnyi, updated.

File: _includes/pbc/all/install-features/202311.0/marketplace/install-the-marketplace-merchant-product-offer-service-points-availability-feature.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+|----------------------------------------------------------------|-------|---------|------------------------------------------------------------------------------------------------------|
+| ProductOfferServicePointAvailabilityRequestItem                | class | created | src/Generated/Shared/Transfer/ProductOfferServicePointAvailabilityRequestItemTransfer                |
+| ProductOfferServicePointAvailabilityResponseItem               | class | created | src/Generated/Shared/Transfer/ProductOfferServicePointAvailabilityResponseItemTransfer               |
+| RestProductOfferServicePointAvailabilityRequestItemsAttributes | class | created | src/Generated/Shared/Transfer/RestProductOfferServicePointAvailabilityRequestItemsAttributesTransfer |
+
+{% endinfo_block %}
+
+## 3) Set up behavior
+
+{% info_block warningBox "Verification" %}
```
[the remark this answers is not in the document]
Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202311.0/marketplace/install-the-marketplace-merchant-product-offer-service-points-availability-feature.md]: @andriitserkovnyi, updated.

**Pull request merged**